docs/md/dev-process.md

```diff
@@ -170,7 +166,10 @@ shared database in `gotc-dev` or *gasp* production.
         thus you could use `pq start` or `pq stop` to easily spin up and turn down
         the db.
 
-3. [Optional] Create wfl DB.
```

Comment:
@ehigham this documentation change doesn't read right to me in the context of setting up a local Postgres, maybe the spec tip should go somewhere else?

Reply:
The code that generates the swagger page doesn't work. I don't know what to do tbh. Anything involving that part of the code makes me tear my hair out so I think your comment is appropriate!

Reply:
Ohhh misunderstanding, I didn't introduce this comment but I thought maybe you did!  Some git weirdness… maybe @tbl3rd ?

Reply:
Got me.  The docs get moved around a lot.

Reply:
Thanks!  I found its rightful home and will fix.